src/Development/IDE/Core/Rules.hs

```diff
@@ -263,6 +264,7 @@ priorityFilesOfInterest = Priority (-2)
 -- and https://github.com/mpickering/ghcide/pull/22#issuecomment-625070490
 getParsedModuleRule :: Rules ()
 getParsedModuleRule = defineEarlyCutoff $ \GetParsedModule file -> do
+    _ <- use_ GetModSummaryWithoutTimestamps file -- Fail if we can't even parse the ModSummary
```

Comment:
Some tests were failing because we reported parse errors twice - once when we tried to parse the `ModSummary`, and once when we tried to parse the actual source. This was the simplest way I could think of to fix it.

Reply:
seems reasonable